pkg/admissioncontroller/webhooks/auth/seed/graph/eventhandler_lease.go

```diff
@@ -1,86 +0,0 @@
-// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Can you also update the graph in the [docs](https://github.com/gardener/gardener/blob/master/docs/deployment/gardenlet_api_access.md), i.e. remove the shade? 😄

Reply:
Thanks for the reminder, I planned to do this but forgot about it. See cc22db5. On the way, I also updated the flow picture and embedded the Excalidraw scene.